lib-injection/auto_inject.rb

```diff
@@ -0,0 +1,56 @@
+if !ENV['skip_autoinject']
+  if system 'skip_autoinject=true bundle show ddtrace'
```

Comment:
`system` will print it's output to STDOUT and STDERR, meaning the user will see the output of all commands in their terminal:
```
$ ruby -e 'puts system "bundle show pry"'
~/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/pry-0.14.2
true
```

I recommend using [`open3`](https://docs.ruby-lang.org/en/2.1.0/Open3.html) to hide or capture STDOUT and STDERR from the commands we run here.
Sometime we might care about them, other times not: it's up to each case.

Here's an example of using it in `ddtrace`: https://github.com/DataDog/dd-trace-rb/blob/2b163dd91c8d349ceb92370c16f4a7be4a691c35/lib/datadog/ci/ext/environment.rb#L501-L511

There are many combinations of methods in `open3`, I recommend using the simplest one for you use case.